src/printer.rs

```diff
@@ -248,11 +248,8 @@ impl<'a, 'b> Printer<'a, 'b> {
         // Don't write too much if we're close to the end
         let width = min(width, (self.output_size.x - start.x) / c.width());
```

Comment:
I think the computation here is wrong: it should be `let repetitions = min(width, (self.output_size.x - start.x)) / c.width();`

Reply:
Yeah, I think you're right.